terraform/public_dns.tf

```diff
@@ -1,8 +1,12 @@
 data "aws_route53_zone" "public_zone" {
+  provider = aws.dns
+
   name = local.public_zone
```

Comment:
The zone is available in the remote state of the `cool-dns-cyber.dhs.gov`along with the role that can modify it. 

See: https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/8#issuecomment-604509738

Reply:
This project is unrelated to the COOL except for using the public DNS.  We will have a lot of projects like this.  Sharing state feels like something that should be done between repositories that are closely related.  We should think carefully about this before proceeding.

Reply:
You are correct @felddy , however we are not currently able assume that COOL role in this (or any other) un-COOL project- here's why:
- We deploy these TeamServers in the `CyHy (Production)` AWS account.  To do so, we must set the `AWS_PROFILE` to our `CyHy (Production)` user profile.
- In order to assume the roles from a COOL remote state, the `AWS_PROFILE` must be set to our `cool-user` credentials.
- Since we cannot have it both ways, the solution is to use the `cool-dns-route53resourcechange-cyber.dhs.gov` profile until these TeamServers can be deployed into a COOL account.  I can add a comment in `providers.tf` to this effect.

Does that work for you?

Reply:
Also, after discussion with @jsf9k, we think it would be best if we created a read-only profile that we use when reading from the remote state:  https://github.com/cisagov/pca-teamserver-aws/blob/improvement/use_cool_dns_zone/terraform/remote_state.tf#L13

Reply:
> * Since we cannot have it both ways, the solution is to use the `cool-dns-route53resourcechange-cyber.dhs.gov` profile until these TeamServers can be deployed into a COOL account.  I can add a comment in `providers.tf` to this effect.

Comment added in https://github.com/cisagov/pca-teamserver-aws/pull/30/commits/7511a5608d7c4a660a9dabd86e4c49fdee2909a5.

Reply:
> Also, after discussion with @jsf9k, we think it would be best if we created a read-only profile that we use when reading from the remote state: https://github.com/cisagov/pca-teamserver-aws/blob/improvement/use_cool_dns_zone/terraform/remote_state.tf#L13

Note that https://github.com/cisagov/cool-accounts/issues/43 has been created for this purpose.